### PR TITLE
Use new 'description' and 'tag' items on list vhosts

### PR DESCRIPTION
**Pull request by dcorbacho**
Part of https://github.com/rabbitmq/rabbitmq-server/issues/2012

### CODE REVIEW

File: lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex [hunk 1]
```diff
@@ -47,4 +58,14 @@ defmodule RabbitMQ.CLI.Ctl.Commands.AddVhostCommand do
...
   def banner([vhost], _), do: "Adding vhost \"#{vhost}\" ..."
+
+  #
+  # Implementation
+  #
+
+  def parse_tags(tags) do
+    String.split(tags, ",", trim: true)
+    |> Enum.map(&String.trim/1)
+    |> Enum.map(&String.to_atom/1)
```
Comment (dumbbell) [inline lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex]: I suppose we can use the existing function in rabbitmq-server

Comment (dumbbell) [reply lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex]: Also same comment as the one attached the other copy of `parse_tags()`: do we really need atoms here?

File: lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex [hunk 2]
```diff
@@ -18,21 +18,32 @@ defmodule RabbitMQ.CLI.Ctl.Commands.AddVhostCommand do
...
   use RabbitMQ.CLI.DefaultOutput
 
-  def usage, do: "add_vhost <vhost>"
+  def usage, do: "add_vhost <vhost> [--description <description> --tags \"<tag1>,<tag2>,<...>\"]"
 
   def usage_additional() do
     [
-      ["<vhost>", "Virtual host name"]
+      ["<vhost>", "Virtual host name"],
+      ["--description <descriptioon>", "Virtual host description"],
```
Comment (dumbbell) [inline lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex]: There is typo in `descriptioon`.

File: lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex [hunk 3]
```diff
@@ -18,21 +18,32 @@ defmodule RabbitMQ.CLI.Ctl.Commands.AddVhostCommand do
...
+      ["--tags <tags>", "Coommand separated list of tags"]
```
Comment (dumbbell) [inline lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex]: There is a typo in `Coommand`. I suppose you mean `Comma` here.

File: lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex (hunk 1 above)
Comment (michaelklishin) [reply lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex]: We agreed to use atoms in rabbitmq/rabbitmq-server#2080. CLI tools try to depend as little as possible on the server modules and when it comes to working with strings, I'd take Elixir over Erlang any day.

**Pull request merged**